src/views/ModelPlan/Discussions/index.tsx

```diff
@@ -348,115 +349,6 @@ const Discussions = ({ modelID, askAQuestion, readOnly }: DiscussionsProps) => {
-              )?.fullName || ''
-            }
-            index={index}
-            className="margin-bottom-2"
-          />
-        )}
-        <span className="margin-left-2 margin-top-05 text-base">
-          {getTimeElapsed(discussion.createdDts)
-            ? getTimeElapsed(discussion.createdDts) + t('ago')
-            : t('justNow')}
-        </span>
-      </div>
-
-      <div
-        className={classNames({
-          'margin-bottom-4': answerQuestion,
-          'mint-discussions__connected': connected,
-          'mint-discussions__not-connected': !connected
-        })}
-      >
-        <p className="margin-y-0 padding-y-1">{discussion.content}</p>
-        <div className="display-flex margin-bottom-2">
-          {/* Rendered a link to answer a question if there are no replies/answers only for Collaborator and Assessment Users */}
-          {hasEditAccess && answerQuestion && (
-            <>
-              <IconAnnouncement className="text-primary margin-right-1" />
-              <Button
-                type="button"
-                unstyled
-                role="button"
-                onClick={() => {
-                  setDiscussionStatusMessage('');
-                  setDiscussionType('reply');
-                  setReply(discussion);
-                }}
-              >
-                {t('answer')}
-              </Button>
-            </>
-          )}
-        </div>
-      </div>
-    </div>
-  );
-
-  const formatDiscussions = (
-    discussionsContent: DiscussionType[],
-    status: DiscussionStatus
-  ) => {
-    if (status === 'ANSWERED') {
-      discussionsContent.sort(sortRepliesByDate); // Sort discusssions by the most recent reply for answered questions
-    }
-
-    return discussionsContent.map((discussion, index) => {
-      return (
-        <div
-          key={discussion.id}
-          className={classNames({
-            'margin-top-4': index > 0,
-            'margin-top-2': index === 0
-          })}
-        >
-          {discussion.replies.length > 0 ? (
-            // If discussions has replies, join together in array for rendering as a connected discussion
-            <div>
-              {[
-                discussion,
-                ...discussion.replies
-              ].map((discussionReply: ReplyType | DiscussionType, replyIndex) =>
-                discussionComponent(
-                  discussionReply,
-                  index,
-                  replyIndex !== discussion.replies.length
-                )
-              )}
-            </div>
-          ) : (
-            // Render only question if no replies
-            discussionComponent(discussion, index, undefined, true)
-          )}
-          {/* Divider to separate questions if not the last question */}
-          {index !== discussionsContent.length - 1 && (
-            <Divider className="margin-top-4" />
-          )}
-        </div>
-      );
-    });
-  };
-
```

Comment:
This function is replaced with `FormatDiscussion` component 

src/views/ModelPlan/Discussions/FormatDiscussion.tsx